pkg/cmd/render/render.go

```diff
@@ -190,6 +195,13 @@ func newTemplateData(opts *renderOpts) (*TemplateData, error) {
 	if err := templateData.setSingleStackIPv6(); err != nil {
 		return nil, err
 	}
+
+	// derive bootstrapIP from local interfaces if not passed by flag
+	if templateData.BootstrapIP == "" {
+		if err := templateData.setBootstrapIP(); err != nil {
```

Comment:
I'd like to have a goal of removing this.  I think it is reasonable for the installer coordinate the bootstrap IP.